features/SPS002_IfcFacility-aggregation.feature

```diff
@@ -0,0 +1,10 @@
+@implementer-agreement
+@SPS
+Feature: SPS002 - IfcFacility aggregation
+The rule verifies that each IfcFacility is a part of an IfcProject, IfcSite or another IfcFacility.
+
+  Scenario: Agreement on each IfcFacility being a part of an IfcProject, IfcSite or another IfcFacility.
+
+      Given A file with Schema Identifier "IFC4X3_TC1" or "IFC4X3_ADD1" or "IFC4X3"
+      And An IfcFacility
+      Then Each IfcFacility must be a part of IfcProject or IfcSite or IfcFacility
```

Comment:
Are the `or`-s really the way to go, or should we rather use `xor` (the real intention in my view)? Is an IfcFacility being part of two or all three of these marked correct as well?